packages/dashboard-core-plugins/src/panels/NotebookPanel.tsx

```diff
@@ -1196,7 +1202,9 @@ class NotebookPanel extends Component<NotebookPanelProps, NotebookPanelState> {
                 sessionLanguage={sessionLanguage}
                 onEditorInitialized={this.handleEditorInitialized}
                 onEditorWillDestroy={this.handleEditorWillDestroy}
-                settings={settings}
+                settings={
+                  settings as editor.IStandaloneEditorConstructionOptions
+                }
```

Comment:
Don't need to cast here if you define the return type on `getSettings` correctly.